### DISCUSSION TITLE
Can't login in Backend

### DISCUSSION THREAD

**Discussion by akiraq** [wintercms/winter#288] [Q&A] [answered]
After install WinterCMS

I can't login in Backend with the account and password (Installation).

I am new in using WinterCMS

Anyone can help me? pls

**Comment by bennothommo** [comment, marked as answer]
@akiraq apologies for missing this.

For future reference if someone encounters this, you have two options - either use the Reset password feature in the Backend login to reset your password and try again, or you can use the `php artisan winter:passwd` command in your terminal to quickly reset the password of an available admin account.

**Reply by akiraq** [reply]
@bennothommo 
Many thanks for your helping ^^"
